go/arch/x86/linux.go

```diff
@@ -10,6 +10,21 @@ import (
 	"github.com/lunixbochs/usercorn/go/models"
 )
 
+const A_PRESENT = 0x80
+const A_DATA = 0x10
```

Comment:
data and code value are wrong, see https://github.com/unicorn-engine/unicorn/pull/876